### ISSUE TITLE
Minimum voltage of the PPS-11360

### ISSUE THREAD

**Issue by michael-dauer** [ap--/voltcraft#24]
I have the same issue as reported by @jonathanlarochelle previously, but with a PPS-11360. To pinpoint the error I connected with a PuTTY serial console to the PPS and issued a 

`VOLT000`

 command to which there was no response indicating an error, whereas to

`VOLT008`

the response was `OK`. I checked the output voltage with `GETD` and as wished it was 0.8 V.

**Comment by michael-dauer** [comment]
I logged the mentioned VOLT??? commands with the PuTTY serial console. The transcript is:

```
GMAX
362700
OK
VOLT000
VOLT007
VOLT008
OK
GETS
008200
OK
VOLT240
OK
GETS
240200
OK
```

Is there anything else I can contribute for the PPS-11360?

**Comment by michael-dauer** [comment]
I closed the issue by mistake, I tried to provide some additional information.

**Comment by jonathanlarochelle** [comment]
Hi,
I have added minimum voltages for the devices I had on-hand (see #22 ).

I would recommend you add the minimum voltage for the PPS-11360 to PPS_MIN_VOLTAGE.

**Comment by ap--** [comment]
Hi @michael-dauer 

Thank you for reporting, and already providing a solution for your powersupply model!
As @jonathanlarochelle suggested, you would just have to change this item in the `PPS_MIN_VOLTAGE` dict:

https://github.com/ap--/voltcraft/blob/19e59f3a3bdbb601ba4549db0ca9c215514dc583/voltcraft/pps.py#L29

From `0` to `0.8` as you've determined.

You can contribute the change by using the github web UI. Click on the edit button, create a new fork, and make the edit, commit the change, and then the github web UI will suggest to create a pull request after.

Let me know if I can help,
Andreas 😃

**Comment by ap--** [comment]
Closed via #25.

I'll prepare a new release right away.
